.github/workflows/csharp-qltest.yml

```diff
@@ -53,7 +53,6 @@ jobs:
         slice: ["1/2", "2/2"]
     steps:
       - uses: actions/checkout@v4
-      - uses: ./.github/actions/fetch-codeql
       - uses: ./csharp/actions/create-extractor-pack
```

Comment:
So this now ensures that the legacy ASP extractor is copied?
Why is that needed?

Reply:
The legacy ASP extractor is needed because it is invoked in the `pre-finalize` script.